src/pages/index.js

```diff
@@ -53,38 +71,87 @@ const profilePopup = new PopupWithForm(
   config.popupProfileSelector,
   (userData) => {
     userInfo.setUserInfo(userData);
-    profilePopup.close();
+    profilePopup.notifyUserSaving(true);
+    api.updateUserInfo(userData).finally(() => {
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**

Reply:
done